src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx

```diff
@@ -0,0 +1,705 @@
+import React from 'react';
+import { expect } from 'chai';
+
+import {
+  mockFetch,
+  resetFetch,
+  setFetchJSONFailure,
+} from 'platform/testing/unit/helpers';
+import environment from 'platform/utilities/environment';
+import { fireEvent, waitFor, within } from '@testing-library/dom';
+import { cleanup } from '@testing-library/react';
+import VAFacilityPage from '../../../../project-cheetah/components/VAFacilityPage';
+import {
+  getVAFacilityMock,
+  getDirectBookingEligibilityCriteriaMock,
+  getClinicMock,
+} from '../../../mocks/v0';
+import {
+  createTestStore,
+  renderWithStoreAndRouter,
+} from '../../../mocks/setup';
+import {
+  mockEligibilityFetches,
+  mockRequestEligibilityCriteria,
+  mockDirectBookingEligibilityCriteria,
+  mockFacilitiesFetch,
+  mockGetCurrentPosition,
+} from '../../../mocks/helpers';
+import { TYPE_OF_CARE_ID } from '../../../../project-cheetah/utils';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCheetah: true,
+  },
+  user: {
+    profile: {
+      facilities: [
+        { facilityId: '983', isCerner: false },
+        { facilityId: '984', isCerner: false },
+      ],
+    },
+  },
+};
+
+const parentSiteIds = ['983', '984'];
+
+const facilityIds = ['983', '983GC', '983GB', '983HK', '983QA', '984'];
+
+const directFacilities = facilityIds.map(id =>
+  getDirectBookingEligibilityCriteriaMock({
+    id,
+    typeOfCareId: TYPE_OF_CARE_ID,
+  }),
+);
+
+const vhaIds = facilityIds.map(
+  id => `vha_${id.replace('983', '442').replace('984', '552')}`,
+);
+
+const facilities = vhaIds.map((id, index) => ({
+  id,
+  attributes: {
+    ...getVAFacilityMock().attributes,
+    uniqueId: id.replace('vha_', ''),
+    name: `Fake facility name ${index + 1}`,
+    lat: Math.random() * 90,
+    long: Math.random() * 180,
+    address: {
+      physical: {
+        ...getVAFacilityMock().attributes.address.physical,
+        city: `Fake city ${index + 1}`,
+      },
+    },
+  },
+}));
+
+const closestFacility = facilities[2];
+closestFacility.attributes.name = 'Closest facility';
+closestFacility.attributes.lat = 39.50603012; // Dayton, OH
+closestFacility.attributes.long = -84.3164749;
+
+describe('VAOS vaccine flow: <VAFacilityPage>', () => {
```

Comment:
These tests are copied and modified versions of the single and multi site V2 tests in the appointment flow.